controller/src/beerocks/master/db/db.cpp

```diff
@@ -3082,6 +3082,19 @@ void db::setting_certification_mode(bool en)
     config.certification_mode = true;
 }
 
+void db::set_bss_info_configuration(const std::string &al_mac, const bss_info_conf_t &bss_info)
```

Comment:
There can be multiple bss_info per al_mac, and you're doing push_back, so this should be called `add_bss_info_configuration`.

Also, to be consistent with `bss_info_conf_t`, I would call it `add_bss_info_conf` (or vice versa, of course).

Reply:
will fix